### PR TITLE
satML-plugin 1.30: third take

### PR DESCRIPTION
**Pull request by OCamlPro-Iguernlala**

### CODE REVIEW

File: packages/satML-plugin/satML-plugin.1.30/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+[
+        ["rm" "%{prefix}%/lib/alt-ergo/plugins/satML-plugin.cmxs"]
+]
+
+depends:
+[
+        "zarith"
+	"camlzip"
+	"ocplib-simplex" {= "0.3"}
+        "alt-ergo" {= "1.30"}
```
Comment (avsm) [inline packages/satML-plugin/satML-plugin.1.30/opam]: please fix the formatting here to be more consistent. (we cannot fix it ourselves due to the request at the top of the file). thanks.

**Comment by avsm** [comment]
note that you can push to the same branch to update a PR. There is no need to keep opening new ones; please see https://help.github.com/articles/about-pull-requests/

File: packages/satML-plugin/satML-plugin.1.30/opam (hunk 1 above)
Comment (OCamlPro-Iguernlala) [inline packages/satML-plugin/satML-plugin.1.30/opam]: Done !

**Comment by OCamlPro-Iguernlala** [comment]
I opened a new PR yesterday because camelus was complaining even after a commit that fixes the issue.

**Pull request merged**